catalog-info.yaml

```
@@ -0,0 +1,18 @@
+# This file records information about this repo. Its use is described in OEP-55:
+# https://open-edx-proposals.readthedocs.io/en/latest/processes/oep-0055-proc-project-maintainers.html
+
+apiVersion: backstage.io/v1alpha1
+kind: Component
+metadata:
+  name: 'event-bus-kafka'
+  description: "Kafka implementation for Open edX event bus."
+  links:
+    - url: "https://github.com/openedx/event-bus-kafka"
+      title: "Event Bus Kafka"
+      icon: "Web"
+  annotations:
+    openedx.org/arch-interest-groups: ""
+spec:
+  owner: group:2u-arch-bom
+  type: 'other'
+  lifecycle: 'production'
```

Can we add the following?:
```suggestion
  lifecycle: 'production'
  dependsOn:
  - 'openedx-events'
```